### PR TITLE
MVG tabular

### LINKED ISSUE #21: Update MVG for KPI data
update client side for https://github.com/vikinganalytics/vibium-cloud/issues/175

- [x] Update python part
- [x] Update Documentation
### Example Data
[Heatpump unlabeled](https://github.com/vikinganalytics/ref-data/blob/master/molnlycke/heatpump/HeatpumpUnlabeled.csv)

### PR DESCRIPTION
**Pull request by OskarLiew**
Updated to reflect changes from the tabular update of vibium-cloud. Tests will not pass before vibium-cloud update

Closes #21

### CODE REVIEW

File: mvg/mvg.py [hunk 1]
```diff
@@ -268,10 +268,35 @@ def create_source(self, sid: str, meta: dict):
...
+        Parameters
+        ----------
+        sid : str
+            source Id
+
+        meta : dict
+            meta information
+
+        columns : List[str]
+            Data variables
```
Comment (FredrikWartenberg) [inline mvg/mvg.py:286]: Mention
- the requirements (timestamp column included and types of remaining). N
- Not possible to update columns later

File: tests/test_api_sources.py [hunk 2]
```diff
@@ -236,19 +253,59 @@ def test_sources_cru_existing(session):
...
+
+
+def test_tabular_measurements_float_timestamps(session):
+    with pytest.raises(HTTPError) as exc:
+        session.create_tabular_measurement(SOURCE_ID_TABULAR, tabular_dict)
+    assert exc.value.response.status_code == 422
+
+
+def test_tabular_measurements(session):
+    columns = tabular_df.columns.tolist()
```
Comment (FredrikWartenberg) [inline tests/test_api_sources.py:289]: Why don't you use a dataset with int timestamps?

**Review by FredrikWartenberg — COMMENTED** [review]
## General
- all fine
- nice example notebook

## Details 
- should there be a test of adding tabular data, or is that tested on the server side?
- in the notebook, this markup does not render correctly: .. note::

File: tests/test_api_sources.py (hunk 2 above)
Comment (OskarLiew) [reply tests/test_api_sources.py:289]: I forgot to change after enforcing int timestamps in vibium-cloud. I will fix this

Comment (OskarLiew) [reply tests/test_api_sources.py:289]: Or no... I remember now. The dataset had timestamps spaced by exactly 1, so I just removed the decimal. I will change to another dataset to make it simpler

File: mvg/mvg.py (hunk 1 above)
Comment (OskarLiew) [reply mvg/mvg.py:286]: timestamp columns is not required when making the source. It will be added regardless.

**Comment by OskarLiew** [comment]
The `.. note::` markup is not rendered in the notebook, but it is rendered in the built documentation. I think it is an okay trade off, since I think more people are going to be copy-pasting from the documentation than download the complete examples

**Review by tuix — APPROVED** [review]

**Pull request merged**